lib/extensions/polyglot_with_database/coursemology/polyglot/language/python/python2_point7.rb

```diff
@@ -0,0 +1,7 @@
+module Extensions::PolyglotWithDatabase::Coursemology::Polyglot::Language::Python::Python2Point7
```

Comment:
I'm a bit concerned about defining new languages in the extensions using meta programming.